build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:jessie
+FROM debian:9.0
```

Comment:
`9.0`? I think that `stretch` or `9` would be better. Debian is *really* conservative changing dependencies.